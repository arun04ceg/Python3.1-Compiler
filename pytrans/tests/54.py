y = 100

def f(x):
 y = 10


f(300)

print(y)

 
