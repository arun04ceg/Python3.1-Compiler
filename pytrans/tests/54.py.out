(program
 (define f (void))
 (define y (void))
 (set-global! y 100)
 (set-global!
  f
  (lambda (x) (call/ec (lambda (return) (let ((y (void))) (set! y 10))))))
 ((get-global f) 300)
 (py-print (get-global y)))
